src/js/components/ToggleGroup/stories/Icons.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Box, ToggleGroup } from 'grommet';
+import { List, MapLocation, Table } from 'grommet-icons';
+
+const optionsIcons = [
+  {
+    icon: <List a11yTitle="List view" />,
+    value: 'list',
+  },
+  {
+    icon: <Table a11yTitle="Map view" />,
+    value: 'Table',
```

Comment:
just lowercasing to be consistent with other options
```suggestion
    value: 'table',
```